src/main/java/com/example/tournamentMaker/tournament/result/ResultService.java

```diff
@@ -99,8 +110,8 @@ private void updateSpecificStatisticInTeam(List<Integer> jerseyNumbersToIncrease
             Optional<FootballPlayer> optionalPlayer = playerRepository.findByJerseyNumberAndTeam(number, team);
             optionalPlayer.ifPresentOrElse(p -> {
                 if (specificStatistic.containsKey(p.getId())) {
-                    int currentValue = specificStatistic.get(p.getId());
-                    specificStatistic.put(p.getId(), ++currentValue);
+                    int increasedValue = specificStatistic.get(p.getId()) + 1;
```

Comment:
dziwny ten zapis. pobierasz get id + 1 ? dziwne mocno

Reply:
kurdiii no tu chodzi o to, żeby licznik danej statystyki zwiększyć o 1
a, że jest to mapa to tak troszkę dziwnie to wygląda

Reply:
po pierwsze nazwij ta lambde player a nie p 
po drugie wyciagnij to do zmiennej a potem zrob inkrementacje i tak bedzie czytelniej